agents/random_agent.py

```diff
@@ -7,9 +7,11 @@ class RandomAgent(Agent):
     An agent that takes legal actions randomly.
     TODO - implement everything
     """
+
     def __init__(self, name):
         super(RandomAgent, self).__init__(name)
 
     def choose_action(self, env):
-        pass
-
+        available_moves = env.available_moves(self)
+        if available_moves:
```

Comment:
And if there aren't any available moves, please return `None`.

Reply:
func without return statement should return automatically "None". Not? Or do you want me to write it manually?